docs/src/css/custom.css

```diff
@@ -291,6 +291,7 @@ h2 {
   --ifm-color-warning-dark: #4f566b;
   --ifm-toc-border-color: rgba(135, 149, 161, 0.2);
   --ifm-table-border-color: rgba(135, 149, 161, 0.2);
+  --ifm-card-background-color: #1a253c;
```

Comment:
@peintnermax I figured out that this is the correct variable for the background color for the autogenerated boxes. I will include it in this pr. Hope that's fine for you!

Reply:
thanks